From 8659f6e98baac8965342b9968cc23d3fc3761416 Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Thu, 2 Jun 2022 12:55:59 -0400
Subject: gcc bugfix try 1

---
 common/bitmap.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/bitmap.cpp b/common/bitmap.cpp
index 895f0ca44c..fc8d828286 100644
--- a/common/bitmap.cpp
+++ b/common/bitmap.cpp
@@ -153,7 +153,7 @@ wxBitmap KiScaledBitmap( BITMAPS aBitmap, wxWindow* aWindow, int aHeight, bool a
     if( aQuantized )
         scale = KiROUND( (double) scale / 4.0 ) * 4;
 
-    SCALED_BITMAP_ID id = { static_cast<BITMAPS>( aBitmap ), scale };
+    SCALED_BITMAP_ID id = { aBitmap, scale };
 
     std::lock_guard<std::mutex> guard( s_BitmapCacheMutex );
     auto it = s_ScaledBitmapCache.find( id );
-- 
2.36.1.windows.1

